src/software/simulation/standalone_simulator.cpp

```diff
@@ -113,50 +113,62 @@ void StandaloneSimulator::setupNetworking(int blue_team_channel, int yellow_team
         boost::bind(&StandaloneSimulator::setBlueTeamDefendingSide, this, _1)));
 }
 
-void StandaloneSimulator::setupInitialSimulationState()
-{
-    RobotState blue_robot_state1(Point(3, 2.5), Vector(0, 0), Angle::half(),
-                                 AngularVelocity::zero());
-    RobotState blue_robot_state2(Point(3, 1.5), Vector(0, 0), Angle::half(),
-                                 AngularVelocity::zero());
-    RobotState blue_robot_state3(Point(3, 0.5), Vector(0, 0), Angle::half(),
-                                 AngularVelocity::zero());
-    RobotState blue_robot_state4(Point(3, -0.5), Vector(0, 0), Angle::half(),
-                                 AngularVelocity::zero());
-    RobotState blue_robot_state5(Point(3, -1.5), Vector(0, 0), Angle::half(),
-                                 AngularVelocity::zero());
-    RobotState blue_robot_state6(Point(3, -2.5), Vector(0, 0), Angle::half(),
-                                 AngularVelocity::zero());
-    std::vector<RobotStateWithId> blue_robot_states = {
-        RobotStateWithId{.id = 0, .robot_state = blue_robot_state1},
-        RobotStateWithId{.id = 1, .robot_state = blue_robot_state2},
-        RobotStateWithId{.id = 2, .robot_state = blue_robot_state3},
-        RobotStateWithId{.id = 3, .robot_state = blue_robot_state4},
-        RobotStateWithId{.id = 4, .robot_state = blue_robot_state5},
-        RobotStateWithId{.id = 5, .robot_state = blue_robot_state6},
-    };
+void StandaloneSimulator::setupInitialSimulationStateDivB()
+{
+    std::vector<RobotStateWithId> blue_robot_states;
```

Comment:
what if we had a function that took the number of robots and made a reasonably spaced configuration?

Reply:
yeah thats probably cleaner, updated